bench/bench.cpp

```diff
@@ -317,11 +317,12 @@ class boost_pool_impl : public any_impl
         string_view s,
         std::size_t repeat) const override
     {
-        parser p;
+        monotonic_resource mr1;
+        parser p(&mr1);
```

Comment:
We shouldn't be doing this , otherwise it won't free the memory when the stack is reallocated. It needs to use the default resource.